src/components/loading.tsx

```diff
@@ -1,3 +1,10 @@
 import React from 'react';
+import styled from 'styled-components';
 
-export const Loading = () : JSX.Element => <span>Loading...</span>;
+const Load = styled.span`
```

Comment:
So this is what you're supposed to do:
- create dirs: `components/loading` that will have `styled.tsx` and `index.tsx` files
- rename `Load` to `Styled` and import it in the `index.tsx` file
- Inside index file, the component name will be `Loading` as it was before

Follow this pattern for all components